rpm/kubelet.spec

```diff
@@ -41,7 +41,7 @@ Command-line utility for interacting with a Kubernetes cluster.
 %package -n kubeadm
 
 Version: 1.5.0
-Release: %{RPM_RELEASE}.alpha.0.1403.gc19e08e
+Release: %{RPM_RELEASE}.alpha.0.1534.gcf7301f
```

Comment:
Ok that should be alright, I will keep an eye out once rpms land and make sure they survive that .alpha in the release, but IIRC that works ok.